### PR TITLE
Converted all check thread logs to ocp diag format

### PR DESCRIPTION
**Pull request by dhawkes**
Internal reference: 274523022

### CODE REVIEW

**Comment by dhawkes** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/10" target="_blank">Graphite</a> 
    * **PR #12** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/12" target="_blank">Graphite</a> 
      * **PR #11** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/11" target="_blank">Graphite</a> 
        * **PR #13** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/13" target="_blank">Graphite</a> 
          * **PR #14** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/14" target="_blank">Graphite</a> 
            * **PR #15** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/15" target="_blank">Graphite</a> 
              * **PR #16** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/16" target="_blank">Graphite</a> 
                * **PR #17** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17" target="_blank">Graphite</a> 
                  * **PR #18** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/18" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/18?utm_source=stack-comment).

**Comment by dhawkes** [comment]
Current dependencies on/for this PR:
* main
  * **PR #10** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/10" target="_blank">Graphite</a> 
    * **PR #12** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/12" target="_blank">Graphite</a> 
      * **PR #11** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/11" target="_blank">Graphite</a> 
        * **PR #13** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/13" target="_blank">Graphite</a> 
          * **PR #14** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/14" target="_blank">Graphite</a> 
            * **PR #15** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/15" target="_blank">Graphite</a> 
              * **PR #16** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/16" target="_blank">Graphite</a> 
                * **PR #17** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/17" target="_blank">Graphite</a> 
                  * **PR #18** <a href="https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/18" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/18?utm_source=stack-comment).

File: src/worker.h [hunk 1]
```diff
@@ -593,6 +593,8 @@ class CheckThread : public WorkerThread {
   // Calculate worker thread specific bandwidth.
   virtual float GetMemoryCopiedData() { return GetCopiedData(); }
 
+  string GetThreadTypeName() { return "Memory Page Check Thread"; }
```
Comment (jcgao1) [inline src/worker.h:596]: Same question about the Getter as in the previous PR.

**Review by jcgao1 — APPROVED** [review]

**Review by jcgao1 — APPROVED** [review]

**Comment by dhawkes** [comment]
@dhawkes started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/18).

**Comment by dhawkes** [comment]
@dhawkes started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/18).

**Comment by dhawkes** [comment]
@dhawkes started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/18).

**Comment by dhawkes** [comment]
@dhawkes started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/opencomputeproject/ocp-diag-sat/18).

**Pull request merged**